src/treq/test/test_response.py

```diff
@@ -91,6 +93,16 @@ def test_json(self):
             self.successResultOf(_Response(original, None).json()),
         )
 
+    def test_json_customized(self):
+        original = FakeResponse(200, Headers(), body=[b'{"foo": ',
+                                                      b'1.0000000000000001}'])
```

Comment:
Why wrap in an object?

Reply:
I'm not sure I understand the question?  This is the pattern followed by all the other tests.